tofu/geom/_core.py

```diff
@@ -4430,10 +4430,16 @@ def _compute_kInOut(self, largs=None, dkwd=None, indStruct=None):
         indout[0, :] = indStruct[indout[0, :]]
         return kIn, kOut, vperp, indout, indStruct
 
-    def compute_dgeom(self, extra=True, plotdebug=True):
+    def compute_dgeom(self, extra=True, show_debug_plot=True):
+        """
+        Computes the dgeom attribute (what does dgeom mean? I'm guessing it stands for shortened dict_geometry?).
```

Comment:
I would say so too. I would let @Didou09 confirm :)
It looks like Florian wrote in the doc some questions, would you mind checking them out (also for next parameter)

Reply:
Do you have any idea what sort of information dgeom contains?